build/TaxJar.nuspec

```diff
@@ -12,20 +12,21 @@
     <iconUrl>https://github.com/taxjar/taxjar.net/raw/master/build/taxjar-icon.png</iconUrl>
     <description>TaxJar makes sales tax filing easier for online sellers and merchants. See local jurisdictional tax reports, get payment reminders, and more. You can use our API to access TaxJar API endpoints, which can get information on sales tax rates, categories or upload transactions.</description>
     <releaseNotes>
-- Update RestSharp to version 106.12.0
+- Update RestSharp to 108.0.3
```

Comment:
Should these be indented or is it a `nuspec` thing?

Reply:
Good question, let me investigate!

Reply:
@saville-stripe I'm not entirely sure on the "why" still, but no indentation seems to have been the standard here, looking through history. I found an example of a previous version with multiple lines and it was the same format: https://github.com/taxjar/taxjar.net/blob/14e329083398873961fa3b36e459bf7f8bf3f688/build/TaxJar.nuspec

Also this stackoverflow is interesting. I guess whitespace doesn't really matter for XML in general, so perhaps it's just a personal preference? https://stackoverflow.com/questions/12526438/importance-of-indenting-xml-file